src/main/java/io/vertx/ext/consul/CheckOptions.java

```diff
@@ -165,6 +168,26 @@ public CheckOptions setHttp(String http) {
     return this;
   }
 
+  /**
+   * Get header map
+   *
+   * @return header map
+   */
+  public Map<String, List<String>> getHeader() {
```

Comment:
minor: should be `getHeaders`/`setHeaders`

Reply:
replaced as requested